client/core/trade_simnet_test.go

```diff
@@ -730,6 +745,50 @@ func (client *tClient) connectDEX(ctx context.Context) error {
 	return nil
 }
 
+// monitorNotificationFeed monitors the client's NotificationFeed for new
+// notifications to prevent the channel from blocking. Notifications received
+// are added to the tClient.notifications slice to be read by consumers
+// subsequently.
+func (client *tClient) monitorNotificationFeed(ctx context.Context) {
+	for {
+		select {
+		case n := <-client.core.NotificationFeed():
```

Comment:
This gets a new feed on each loop iteration.

Reply:
Oops. Thanks for that catch!